### PR TITLE
fix: Allow overriding scheme for container probes

### PR DESCRIPTION
**Pull request by keelerm84**
Our liveness and readiness probes will default to HTTP. However, if a
customer wishes to enable TLS on the relay, those checks will need to be
HTTPS.

To support this option, the probe configuration will be made
configurable with our existing behavior as the default. Customers can
override the scheme if they wish using

`relay.livenessProbe.httpGet.scheme = "HTTPS"`
`relay.readinessProbe.httpGet.scheme = "HTTPS"`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ Here's a summary of the available configuration options:
 | relay.environmentVariables                    | array   | `[]`                                                         | Defines container environment variables to configure the Relay Proxy instance (via container spec) |
 | relay.secrets                                 | array   | `[]`                                                         | Defines container environment variables or volumes populated from k8s secrets                      |
 | relay.volume                                  | object  | `{}`                                                         | Enables offline mode or references an existing config file from a defined volume                   |
+| relay.livenessProbe                           | object  | `{httpGet: { port: "api",  path: "/status" }}`               | Defines the liveness probe for the relay container                                                 |
```
Comment (louis-launchdarkly) [inline README.md:46]: Do we need documentation that the scheme can be changed, or this is standard for httpGet already so developers would already know what to do?

**Review by louis-launchdarkly — APPROVED** [review]
The change itself looks good, have a non-blocking comment about documentation.

**Review by louis-launchdarkly — APPROVED** [review]
The change itself looks good, have a non-blocking comment about documentation.

File: README.md (hunk 1 above)
Comment (keelerm84) [reply README.md:46]: httpGet is a defined liveness configuration in k8s. So I think it will be well-known.

**Pull request merged**